pkg/config/template.go

```diff
@@ -80,6 +83,7 @@ const (
 	crioNetworkConfig templateGroup = 5
 	crioMetricsConfig templateGroup = 6
 	crioTracingConfig templateGroup = 7
+	crioStatsConfig   templateGroup = 8
```

Comment:
Can simplify this to use iota. https://github.com/cri-o/cri-o/pull/5469

Reply:
I've rebased on the pkg/config commit